### ISSUE TITLE
vncserver leave socket in /tmp/.X11-unix and auto increasing display num

### ISSUE THREAD

**Issue by Benik3** [TigerVNC/tigervnc#912]
Hello.
I found that VNCserver leave orphan socket in /tmp/.X11-unix/ when the vncserver fail to start (e.g. due to SELinux and creating PID file) or if PC shut down e.g. because of power lost.
This file (X1, X2...) is not removed by vncserver -kill and what's worse, when I have service which should run e.g. Display :1, if it fails because of the orphan X1, it automatically increase the number and try :2. But this crash the next service which should create :2 display (I have multiple services for different users).

Is there proper way how to remove this file and ideally also how to block vncserver to try automatically next display?
I'm testing on CentOS 8 (non stream)

EDIT: It looks like with .X1-lock it's the same problem
EDIT2: Is there any other "lock"? I still get that "A VNC server is already running as :1", even that there is no display in vncserver -list and there is no /etc/.X11-unix/X1 nor /etc/.X1-lock :/

Thank you

**Comment by CendioOssman** [comment]
Our `vncserver` does not try a different display like that. I'm afraid that's a modification by your distribution that you'll have to report to them.

As for the lock, we check the following things:

 * The X11 TCP socket (port 6000 + display)
 * The VNC TCP socket (port 5900 + display)
 * The X11 Unix socket (`/tmp/.X11-unix/X<display>`)
 * `/tmp/.X<display>-lock`

All of these need to be available for `vncserver` to start a new server. So in your case there might be something blocking the TCP ports? Please use `ss` to check for that.

**Comment by Benik3** [comment]
Thanks for the reply.
It was probably opened socket, after reboot of system it was OK.
I tested more the problem with auto increasing of the display number and not, it's not problem only with this package.
I just tested with CentOS7 with the same result, then I tried to download and compile 1.10.0 tigervnc from source and it still act the same (I wasn't able to get the service to work with this manual build but it happens also with "standard" way). 
![obrazek](https://user-images.githubusercontent.com/4532647/70481113-df079000-1ae1-11ea-9e28-75011552daf3.png)
![obrazek](https://user-images.githubusercontent.com/4532647/70481281-60f7b900-1ae2-11ea-8f60-71ab2c72576c.png)
